packages/terra-show-hide/docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Terra Show Hide
+
+Show Hide component that will show a preview of content or show the full content with the click of a button.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-show-hide`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ShowHide from 'terra-show-hide';
+
+  <ShowHide preview="Lorem ipsum dolor sit amet">
+    {Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.}
+  </ShowHide>
+```
+
+## Component Features
+
+<!-- Uncomment supported features.
```

Comment:
Add the supported features & update links to point to terra-ui: https://engineering.cerner.com/terra-ui/#/getting-started/terra-ui/features

Reply:
Which features will be supported?

Reply:
@emilyrohrbough https://github.com/cerner/terra-core/pull/1910/commits/d21cec6ba55386326e3b4650a3dbb95935e9c0e6